manifests/init.pp

```diff
@@ -155,18 +156,19 @@
 
   include foreman_proxy_content::pub_dir
 
-  if $qpid_router {
-    class { 'foreman_proxy_content::dispatch_router':
-      agent_addr    => $qpid_router_agent_addr,
-      agent_port    => $qpid_router_agent_port,
-      ssl_ciphers   => $qpid_router_ssl_ciphers,
-      ssl_protocols => $qpid_router_ssl_protocols,
-      logging_level => $qpid_router_logging_level,
-      logging       => $qpid_router_logging,
-      logging_path  => $qpid_router_logging_path,
-    }
-    contain foreman_proxy_content::dispatch_router
+  class { 'foreman_proxy_content::dispatch_router':
+    agent_addr    => $qpid_router_agent_addr,
+    agent_port    => $qpid_router_agent_port,
+    ssl_ciphers   => $qpid_router_ssl_ciphers,
+    ssl_protocols => $qpid_router_ssl_protocols,
+    logging_level => $qpid_router_logging_level,
+    logging       => $qpid_router_logging,
+    logging_path  => $qpid_router_logging_path,
+    ensure        => bool2str($enable_katello_agent, 'present', 'absent'),
```

Comment:
Odd: I thought there was a linting rule that preferred `ensure` to be the first line if given.